src/eip7594/eip7594.c

```diff
@@ -353,35 +353,34 @@ static void deduplicate_commitments(
 }
 
 /**
- * Compute random linear combination challenge scalars for verify_cell_kzg_proof_batch. In this, we
- * must hash EVERYTHING that the prover can control.
+ * Compute the challenge value used for batch verification of cell KZG proofs.
  *
- * @param[out]  r_powers_out        The output challenges, length `num_cells`
+ * @param[out]  challenge_out       The output challenge as a BLS field element
  * @param[in]   commitments_bytes   The input commitments, length `num_commitments`
  * @param[in]   num_commitments     The number of commitments
  * @param[in]   commitment_indices  The cell commitment indices, length `num_cells`
  * @param[in]   cell_indices        The cell indices, length `num_cells`
- * @param[in]   cells               The cell, length `num_cells`
- * @param[in]   proofs_bytes        The cell proof, length `num_cells`
+ * @param[in]   cells               The cells, length `num_cells`
+ * @param[in]   proofs_bytes        The cell proofs, length `num_cells`
  * @param[in]   num_cells           The number of cells
  */
-static C_KZG_RET compute_r_powers_for_verify_cell_kzg_proof_batch(
-    fr_t *r_powers_out,
+C_KZG_RET compute_verify_cell_kzg_proof_batch_challenge(
+    fr_t *challenge_out,
     const Bytes48 *commitments_bytes,
     size_t num_commitments,
     const uint64_t *commitment_indices,
     const uint64_t *cell_indices,
     const Cell *cells,
     const Bytes48 *proofs_bytes,
-    uint64_t num_cells
+    size_t num_cells
```

Comment:
Hm, was this change done? I seem to recall all public functions should take as input uint64_t. I guess this is not really a public func, but still, why change it from uint64_t to size_t?